packages/ignition/tailwind.config.ts

```diff
@@ -1,14 +1,11 @@
 import type { Config } from "tailwindcss";
 import plugin from "tailwindcss/plugin";
 import icons from "rocketicons/tailwind";
+import { tailwind as codeBlock } from "@rocketclimb/code-block";
```

Comment:
LOL. I think we should give it a more specific name, unless '_tailwind_' is a standard for Tailwind plugins.